src/astro/sql/operators/agnostic_load_file.py

```diff
@@ -161,6 +169,9 @@ def load_file(
     :type file_conn_id: str
     :param task_id: task id, optional.
     :type task_id: str
+    :param normalize_config: config dict for pandas json_normalize method
+        https://pandas.pydata.org/docs/reference/api/pandas.json_normalize.html
```

Comment:
I agree it's an implementation detail but since we are just using their function I think it would be wise to expose it since we are not providing a subset of features or modifying them in any way.